packages/frontend-2/components/dashboard/Blog/Card.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <NuxtLink :to="webflowItem.url" target="_blank">
+    <div
+      class="bg-foundation border border-outline-3 rounded-xl flex flex-col overflow-hidden hover:border-outline-5 transition"
+    >
+      <NuxtImg
```

Comment:
What's NuxtImg do for us in this scenario? Does it somehow optimize the links?

Reply:
@andrewwallacespeckle

>NuxtImg creates variants of the image, and will serve the best one, depending on device and connection quality.

How and when does it do this?

Reply:
My mistake, it doesn't create variants, it optimises on-the-fly when they're requested, applying resizing and compression based on the device and network. 

Some googling suggests this happens server-side when the image is first accessed, with results cached for subsequent requests.

I can use the plain old img tag here if you want?

Reply:
@andrewwallacespeckle I'm just curious how it actually works, I do think its a beneficial plugin, but it has to be configured correctly. That said, its probably better than nothing so lets see how this performs live.